File: ts/iq/w-pay~set~account+custom_payment_method.ts
/** <iq type="set" xmlns="w:pay">. Replies with one of: Success, IQErrorWithCodeAndReason. */
export function buildBrPaymentCreateCustomPaymentMethod(deviceId: string): BinaryNode {
  return { tag: 'iq', attrs: { type: 'set', to: 's.whatsapp.net', xmlns: 'w:pay' }, content: [
      { tag: 'account', attrs: { action: 'create-custom-payment-method', device_id: deviceId, country: 'BR' } },
    ] }
}

File: go/iq/w-pay~set~account+custom_payment_method.go
// BuildBrPaymentCreateCustomPaymentMethod builds <iq type="set" xmlns="w:pay">. Replies with one of: Success, IQErrorWithCodeAndReason.
func BuildBrPaymentCreateCustomPaymentMethod(deviceId string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"type": "set", "to": "s.whatsapp.net", "xmlns": "w:pay"},
		Content: []waBinary.Node{
			{Tag: "account", Attrs: waBinary.Attrs{"action": "create-custom-payment-method", "device_id": deviceId, "country": "BR"}},
		},
	}
}